### ISSUE TITLE
Variable speed cooling tower model needs a bypass control option 

### ISSUE THREAD

**Issue by Nigusse** [NREL/EnergyPlus#7718]
Issue overview
--------------
Please change this line to a description of the defect, with useful supporting information including how the issue was found, conditions that trigger this defect, workarounds for users of the current version, and a clear description of how to validate that a fix is complete for this defect.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

#### Additional details
A user file has a condenser loop connected to large number of water source heat pumps providing cooling and heating to thermal zones. The condenser loop minimum and maximum temperatures were controlled (maintained) using a boiler and a cooling tower. The boiler and the cooling tower were placed in series on a supply branch the of this condenser loop. The boiler first and followed the variable speed cooling tower. When the condenser loop temperature drops below the boiler turns ON and heats the water to 20C. This water at 20C has to pass the VS cooling tower and the cooling tower fan is OFF because the water is below its setpoint temperature of 30C but the tower provides free cooling, which over cools the water below 20C. For this reason, it is suggested to add a ***tower bypass control option*** to the variable speed cooling tower model. See the pull request for issue #7464 for further comments.

**Comment by Nigusse** [comment]
Looking at the cooling tower objects IDD, of the three cooling tower models in EnergyPlus listed below only *CoolingTower:SingleSpeed* has ***Capacity Control*** input field.
- CoolingTower:SingleSpeed
- CoolingTower:TwoSpeed
- CoolingTower:VariableSpeed

The single speed cooling tower *Capacity Control* input field has two choice key shown below:
     A11, \field Capacity Control
         \type choice
         \key FanCycling
         \key FluidBypass
         \default FanCycling
 
The other two cooling tower model need this input filed to be able to avoid overcooling during free-cooling operating mode. @mjwitte any comment on this?

**Comment by mjwitte** [comment]
@Nigusse The TwoSpeed and VariableSpeed towers inherently have some capacity control with fan speeds and fan off. The goal here is to add FluidBypass as an option for low loads when the fan is off. So, it doesn't seem that the same choices would apply. It's worth investigating how these towers are typically controlled in the field. It may be that bypass is standard and we don't even need an input choice?

**Comment by Nigusse** [comment]
I agree it is not a capacity control. I was looking at the single speed cooling tower input field, the capacity control is not as such a capacity control, may be it is during FanCycling mode but even then it is used at low load only, I think. You are right we do not need input field to implement the bypass operating mode. We can always trigger bypass whenever tower leaving water temperature at fan off (free cooling mode) is less than the setpoint. May be there are other conditions that favor bypass control, I have think about it.

**Comment by rraustad** [comment]
That's a form of free cooling, as long as the tower is on. I agree with @mjwitte. We need to know what manufacturers do with tower water flow when no cooling is needed when series configuration is used. Do manufacturers have special considerations when equipment is in series with towers? Sounds like you need to make a phone call.
